help_data.yaml

```diff
@@ -76,3 +66,25 @@ livepatch:
       livepatches cannot be enabled on fips-enabled systems. You can find out
       more about Ubuntu Kernel Livepatch service at
       https://ubuntu.com/security/livepatch
+
+ros:
+  help: |
+    ros provides access to a private PPA which includes available high
+    and critical CVE fixes for Ubuntu Robot Operating System (ROS) packages
+    between the end of the standard Ubuntu LTS security maintenance and its
+    end of life. For full access to ROS ESM and security fixes, both
+    esm-infra and esm-apps services will also be enabled. To get additional
+    bug fixes, enable ros-updates in addition to ros.
+    You can find out more about the ROS ESM service at
+    https://ubuntu.com/robotics/ros-esm
+
+ros-updates:
+  help: |
+    ros-updates provides access to a private PPA which includes
+    non-security-related bugfixes for Ubuntu Robot Operating System (ROS)
+    packages between the end of the standard Ubuntu LTS security maintenance
+    and its end of life. For full access to ROS ESM and security fixes, both
+    esm-infra and esm-apps services will also be enabled. To get the
+    security fixes, enable ros in addition to ros-updates.
+    You can find out more about the ROS ESM service at
+    https://ubuntu.com/robotics/ros-esm
```

Comment:
```suggestion
    ros-updates provides access to a private PPA which includes
    non-security-related updates for Robot Operating System (ROS)
    packages between the end of the standard Ubuntu LTS security maintenance
    and its end of life. For full access to ROS ESM, security and non-security updates,
    both esm-infra and esm-apps services will also be enabled.
    You can find out more about the ROS ESM service at
    https://ubuntu.com/robotics/ros-esm
```

Reply:
Removed `To get the security fixes, enable ros in addition to ros-updates.` since enabling `ros-updates` should also automatically enable `ros`.
Maybe this should be mentioned also in this doc.

Reply:
I also find the following difficult to understand, it could use some rewording:
`between the end of the standard Ubuntu LTS security maintenance and its end of life.`